src/app/components/game/game.component.ts

```diff
@@ -118,31 +125,68 @@ export class GameComponent implements OnInit, OnChanges {
         }
         this.drawBoard();
     };
-    private _mouseDownEventListener: Function = () => {
+    private _mouseDownEventListener: Function = (e) => {
         // condition when not on latest move
         if (this.displayedMoveIndex !== this.game.moveHistory.length) {
             return;
         }
         // when mouse is pressed down
         this.CURSOR_DATA.mouseIsDown = true;
+        // left click
+        if (e.which === 1) {
+            this.CURSOR_DATA.mouseLeftIsDown = true;
+            this.CURSOR_DATA.mouseRightIsDown = false;
+            // right click
+        } else if (e.which === 3) {
+            this.CURSOR_DATA.mouseRightIsDown = true;
+            this.CURSOR_DATA.mouseLeftIsDown = false;
+        }
```

Comment:
We want to put the comments here inside the conditionals, otherwise they look wrong (I fixed this when I made https://github.com/JBader89/schackmatt/pull/1)